src/StripeEmitter.ts

```diff
@@ -0,0 +1,51 @@
+/** @private
+ * (For internal use in stripe-node.)
+ * Wrapper around the Event Web API. */
+class _StripeEvent extends Event {
+  data: any;
+  constructor(eventName: string, data: any) {
+    super(eventName);
+    this.data = data;
+  }
+}
+
+/** Minimal EventEmitter wrapper around EventTarget. */
+class StripeEmitter {
+  eventTarget: EventTarget;
+  listenerMapping: Record<string, (...args: any[]) => any>;
+
+  constructor(eventTarget: EventTarget) {
+    this.eventTarget = eventTarget;
+    this.listenerMapping = {};
+  }
+
+  on(eventName: string, listener: (...args: any[]) => void): void {
+    const listenerWrapper = (event: any): void => {
```

Comment:
can we have a better type than any?

Reply:
Changed to `_StripeEvent` in https://github.com/stripe/stripe-node/pull/1668/commits/fafb866900d20f3001c12537d12305df140e8f20